helm_deploy/hmpps-assessments-api/templates/_envs.tpl

```diff
@@ -16,6 +16,9 @@ env:
   - name: OAUTH_ENDPOINT_URL
     value: "{{ .Values.env.OAUTH_ENDPOINT_URL }}"
 
+  - name: COURT_CASE_API_BASE_URL
+    value: "{{ .Values.env.COURT_CASE_API_BASE_URL }}"
+
```

Comment:
Delete the env var below, this is my mistake when I originally created the config

Reply:
PSR have a similar config setting in their applications. I think we can leave it in.